securityfilter.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/shurcooL/githubv4"
+)
+
+func filterSecurityPullRequests(
+	client *githubv4.Client,
+	pullRequests *[]pullRequest,
+) ([]pullRequest, error) {
+	filteredPullRequests := []pullRequest{}
+	for _, pr := range *pullRequests {
+		type vulnQuery struct {
+			Repository struct {
+				VulnerabilityAlerts struct {
+					Nodes []struct {
+						VulnerableRequirements string
+						State                  string
+						SecurityVulnerability  struct {
+							Package struct {
+								Name string
+							}
+						}
+					}
+				} `graphql:"vulnerabilityAlerts(first:100,states:OPEN)"`
+			} `graphql:"repository(owner: $owner, name: $name)"`
+		}
+		vulnVars := map[string]interface{}{
+			"owner": githubv4.String(pr.owner),
+			"name":  githubv4.String(pr.repository),
+		}
+		var vulnQ vulnQuery
+
```

Comment:
typically avoid blank lines in small-ish functions since it's a bit of a personal preference where to put them - easiest way to make it consistent no matter who writes it: don't add 'em :relaxed:

Reply:
Doesn't that just mean the ones with the preference to have no blank lines win? :thinking: :smile: 

Anyways, updated as requested!